apps/web-giddh/src/app/new-inventory/component/inventory-product-service-list/inventory-product-service-list.component.html

```diff
@@ -1,86 +1,201 @@
-                                    </div>
-                                </td>
-                                <td data-title="Name">
-                                    <div class="product-name-image d-flex">
-                                        <div>
-                                            <img src="{{ imgPath }}images/add-image.svg" />
-                                        </div>
-                                        <div class="cursor-pointer mx-2">
-                                            <a
-                                                href="javascript:void(0)"
-                                                [routerLink]="['/pages/new-inventory/about-product-service-detail']"
-                                            >
-                                                <p>Product Name</p>
-                                                <p class="text-gray">(34 Items)</p>
-                                            </a>
-                                        </div>
-                                    </div>
-                                </td>
-                                <td data-title="Type">Product</td>
-                                <td data-title="Group">Group 1</td>
-                                <td data-title="SKU">23124</td>
-                                <td data-title="HSN/SAC">12345</td>
-                                <td data-title="Account/s">
-                                    <div>Cost of goods sold</div>
-                                    <div>Sales 2</div>
-                                </td>
-                                <td data-title="Pur Rate">₹10,00,000.00</td>
-                                <td data-title="Sale Rate">₹10,00,000.00</td>
-                                <td data-title="Closing Qty">21,200 nos.</td>
-                            </tr>
-                        </tbody>
-                    </table>
+                <div class="total-details-box border-none due-box">
+                    <h5>
+                        Item Count
+                        <div class="mt-3">
+                            <span class="d-inline-flex number-total"> 23 </span>
+                        </div>
+                    </h5>
                 </div>
             </div>
         </div>
     </div>
+    <div class="table-responsive">
+        <table mat-table [dataSource]="dataSource" class="table giddh-table on-mobile-view">
+            <ng-container matColumnDef="select">
+                <th mat-header-cell *matHeaderCellDef>
+                    <mat-checkbox
+                        color="primary"
+                        class="ml-3"
+                        (change)="$event ? masterToggle() : null"
+                        [checked]="selection.hasValue() && isAllSelected()"
+                        [indeterminate]="selection.hasValue() && !isAllSelected()"
+                        [aria-label]="checkboxLabel()"
```

Comment:
Avoid setting property with method call in template as this method will be executed on every change detection impacting the performance.